packages/hardhat-etherscan/src/network/prober.ts

```diff
@@ -20,30 +20,31 @@ export async function getEtherscanEndpoints(
   }
 
   const chainIdsToNames = new Map(
-    Object.entries(chainConfig).map(([mapName, config]) => [
+    entries(chainConfig).map(([chainName, config]) => [
```

Comment:
Created an `entries` function that has the proper types. I think we also have this function somewhere in `hardhat-core`, and probably in other places too. At some point, it might be useful to have a `hardhat-common` published package, with utils and common config.